app/auth/login/page.tsx

```diff
@@ -58,18 +59,13 @@ export default function LoginPage() {
                     </div>
 
                     <div>
-                        <button
-                            type="submit"
-                            className="flex w-full justify-center rounded-md bg-indigo-600 px-3 py-1.5 text-sm/6 font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
-                        >
-                            Sign in
-                        </button>
+                        <LoginFormSubmitState />
                     </div>
                 </form>
 
                 <p className="mt-10 text-center text-sm/6 text-gray-500">
-                    Not a member?{' '}
-                    <a href="localhost:3000/register" className="font-semibold text-indigo-600 hover:text-indigo-500">
+                    Not a user yet?{' '}
+                    <a href={process.env.REGISTRATION_URL} className="font-semibold text-indigo-600 hover:text-indigo-500">
```

Comment:
instead of putting `localhost:3000/register` as a `.env` variable, a better alternative is to split it. so it would look something like this

`BASE_URL=http://localhost:3000`
this is the .env file.

in this file you can change it to

**<a href=`${process.env.BASE_URL/register`**

this way, we can use this `BASE_URL` everywhere